src/swagger-specs/sandbox.yaml

```diff
@@ -614,12 +662,15 @@ definitions:
         example: 180e1210-d33a-11e9-8b17-d9e538719ued
       name:
         type: string
+        description: The identifier that will be used to access the sandbox in future requests. This value must be unique, and best practice is to make it as descriptive as possible. This value cannot contain any spaces or special characters.
```

Comment:
As this is human generated. It should be mentioned explicitly.  Although, this is a description for the response so guidance on name selection is unnecessary. it needs reframing as 'The user generated human friendly name for the sandbox that will be used ...'
The creation description should be in 'CreateSandboxPayload'
This then becomes a rabbit hole of fixing the swagger docs 😂 
also, this ref is poorly named, ha.